### PR TITLE
Allow for custom container element

### PR DESCRIPTION
**Pull request by thibmaek**
> The `<aside>` HTML element represents a portion of a document whose content is only indirectly related to the document's main content. Asides are frequently presented as sidebars or call-out boxes.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside

Not sure why we would not use <aside> for the container element to indicate to screen readers that cookie though is not part of the main content on the website

### CODE REVIEW

**Comment by LukasCornille** [comment]
Seems like a good idea! Could you add this new config variable to the [docs](https://github.com/inthepocket/cookie-though/blob/master/docs/jekyll/configuration.md) and the [types](https://github.com/inthepocket/cookie-though/blob/master/src/types.ts) and validate code coverage?

**Comment by thibmaek** [comment]
For you I will do this @LukasCornille ! I already updated the types, and did the docs just now.

**Comment by LukasCornille** [comment]
Looking good! Just the the tests appear to be failing at the moment 🙈.

**Review by LukasCornille — APPROVED** [review]

**Comment by LukasCornille** [comment]
@thibmaek I've added 'aside' as the default element in case it's undefined in the defined config ;). 
Go ahead, so this has my stamp of approval peepoHappy

**Comment by gboston** [comment]
Is there a reason to make this configurable?

File: docs/jekyll/configuration.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ The config object consists of (? means it's optional):
 * [header](#header)
 * [customizeLabel](#customizelabel): string
 * [cookiePolicy](#cookiepolicy)?
+* [containerElement](#containerElement)?
```
Comment (gboston) [inline docs/jekyll/configuration.md:25]: I would not allow to use other container elements.

File: src/lib.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ const ee = new EventEmitter();
 
 export const configure = (conf: Config) => {
   config = conf;
-  const container = document.createElement('div');
+  const container = document.createElement(config.containerElement ?? 'aside');
```
Comment (gboston) [inline src/lib.ts:19]: Just use aside.

**Review by gboston — CHANGES_REQUESTED** [review]

**Comment by LukasCornille** [comment]
> Is there a reason to make this configurable, seems to be best practice to use aside?

This seems like a good idea, didn't think about this yet. I'll implement the changes.

**Review by gboston — APPROVED** [review]
lgtm

**Review by PieterJanVdb — APPROVED** [review]

**Pull request merged**